source/fundamentals/enterprise-auth.txt

```diff
@@ -0,0 +1,216 @@
+      connection string as a parameter using the format:
+      ``<PROPERTY_NAME>:<value>``.
+
+      Your code to instantiate a ``MongoClient`` using GSSAPI and additional
+      properties might resemble the following:
+
+      .. code-block:: java
+
+         MongoClient mongoClient = MongoClients.create("<username>@<hostname>:<port>/?authSource=<authenticationDb>&authMechanism=GSSAPI&SERVICE_NAME:myService");
+
+   .. tab::
+      :tabid: MongoCredential
+
+      To specify one of the GSSAPI additional properties, call the
+      ``withMechanismProperty()`` method on your ``MongoCredential``
+      instance and pass the property name and value as parameters. Use the
+      property name constants defined in the ``MongoCredential`` class:
+
+      - :java-docs:`SERVICE_NAME_KEY </apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#SERVICE_NAME_KEY>`
+      - :java-docs:`CANONICALIZE_HOST_NAME_KEY </apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#CANONICALIZE_HOST_NAME_KEY>`
+      - :java-docs:`JAVA_SUBJECT_KEY </apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#JAVA_SUBJECT_KEY>`
+      - :java-docs:`JAVA_SASL_CLIENT_PROPERTIES_KEY </apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#JAVA_SASL_CLIENT_PROPERTIES_KEY>`
+
+      Your code to instantiate a ``MongoClient`` using GSSAPI and additional
+      properties might resemble the following:
+
+      .. include:: /includes/fundamentals/code-snippets/auth-credentials-gssapi-properties.rst
+
+.. note::
+
+   On Windows, Oracle’s JRE uses `LSA <https://msdn.microsoft.com/en-us/library/windows/desktop/aa378326.aspx>`__
+   rather than `SSPI <https://msdn.microsoft.com/en-us/library/windows/desktop/aa380493.aspx>`__
+   in its implementation of GSSAPI which limits interoperability with
+   Windows Active Directory and implementation of single sign-on. See the
+   following articles for more information:
+
+   - `JDK-8054026 <https://bugs.openjdk.java.net/browse/JDK-8054026>`__
+   - `JDK-6722928 <https://bugs.openjdk.java.net/browse/JDK-6722928>`__
+   - `SO 23427343 <https://stackoverflow.com/questions/23427343/cannot-retrieve-tgt-despite-allowtgtsessionkey-registry-entry>`__
+
+
+.. _plain-auth-mechanism:
+
+LDAP (PLAIN)
+~~~~~~~~~~~~
+
+*Available in MongoDB Enterprise Edition 3.4 and later.*
+
+You can authenticate to a Lightweight Directory Access Protocol (LDAP)
+server using your directory server username and password (if required).
+
+.. note::
+   The authentication mechanism is named ``PLAIN`` instead of ``LDAP`` since it
+   authenticates using the `PLAIN Simple Authentication and Security Layer
+   (SASL) defined in RFC-4616 <https://tools.ietf.org/html/rfc4616>`_.
+
+You can specify this authentication mechanism by setting the ``authMechanism``
+parameter to ``PLAIN`` and including your LDAP username and password in the
+:manual:`connection string </reference/connection-string/>` as shown
+in the following sample code.
```

Comment:
```suggestion
:manual:`connection string </reference/connection-string/>`.
```
the words `following` and `code` are used pretty often in these next few paragraphs. Not sure the context is needed here.